docs/content/doc/usage/packages/rpm.en-us.md

```diff
@@ -79,7 +79,7 @@ The server responds with the following HTTP Status codes.
 
 ## Delete a package
 
-To delete a Debian package perform a HTTP DELETE operation. This will delete the package version too if there is no file left.
+To delete an RPM package perform a HTTP DELETE operation. This will delete the package version too if there is no file left.
```

Comment:
```suggestion
To delete a RPM package perform a HTTP DELETE operation. This will delete the package version too if there is no file left.
```

Reply:
`an` should be correct here as RPM is pronounced _/ɑː/ /piː/ /ɛm/_
The pronunciation defined if `a` or `an` is used: https://www.grammarly.com/blog/indefinite-articles-a-and-an/